aggregators/codec.go

```diff
@@ -474,7 +474,9 @@ func (gl *GlobalLabels) ToProto() *aggregationpb.GlobalLabels {
 
 	// Keys must be sorted to ensure wire formats are deterministically generated and strings are directly comparable
 	// i.e. Protobuf formats are equal if and only if the structs are equal
-	pb.Labels = make([]*aggregationpb.Label, 0, len(gl.Labels))
+	if len(gl.Labels) > cap(pb.Labels) {
+		pb.Labels = make([]*aggregationpb.Label, 0, len(gl.Labels))
+	}
```

Comment:
**question**: should we slice the labels here and below ? 

`pb.Labels = pb.Labels[:0]`

Reply:
Good question. Slices from VT pool should be either nil or already resliced with [:0]. It should not be needed here.